.gitattributes

```diff
@@ -0,0 +1 @@
+js/deps/pluralize.js binary
```

Comment:
We probably don't need this, as we don't actually want to ship it... just copy it.